OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -9,13 +9,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt> 
                 <title xml:lang="gez" xml:id="title1">ሰይፈ፡ ሥላሴ፡, መልክአ፡ ሥላሴ፡, ሰቆቃወ፡ ድንግል፡, ነዓ፡ ኀቤየ፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡, መልክአ፡ ሥላሴ፡</title>
                 <title xml:lang="gez" corresp="#title1" type="normalized">Sayfa Śǝllāse, Malkǝʾa Śǝllāse, Saqoqāwa dǝngǝl, Naʿā ḫabeya ʾo-ʾǝgziʾǝya ʾIyasus Krǝstos, Malkǝʾa Śǝllāse</title>
-                <title xml:lang="en" corresp="#title1">Sword of the Trinity, Image of the Trinity, Lament on the Virgin, ‘Come to me, o my Lord Jesus Christ’, Malkǝʾ-hymn <!-- Malkǝʾ in italics --> to the Trinity, Prayer to the Three Names</title>
+                <title xml:lang="en" corresp="#title1">Sword of the Trinity, Malkǝʾ-hymns <!-- Malkǝʾ in italics --> to the Trinity, Lament on the Virgin, ‘Come to me, o my Lord Jesus Christ’, Prayer to the Three Names</title>
```

Comment:
Lament OF the Virgin?

Reply:
Both EAE and LIT2285Saqoqa have "on". I would stick to that.

Reply:
Ah, ok! I supposed it was just a typo, but let's keep it then :)